data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -172,20 +172,24 @@ else
     IRONIC_HOST="${IRONIC_IP}"
 fi
 
+# Create a podman secret for the image-customization-server 
+sudo podman secret rm pull-secret || true
+base64 /root/.docker/config.json | sudo podman secret create pull-secret -
+
 # Embed agent ignition into the rhcos live iso
 sudo podman run -d --net host --privileged --name image-customization \
     --env DEPLOY_ISO="/shared/html/images/ironic-python-agent.iso" \
     --env DEPLOY_INITRD="/shared/html/images/ironic-python-agent.initramfs" \
     --env IRONIC_BASE_URL="http://${IRONIC_HOST}" \
     --env IRONIC_RAMDISK_SSH_KEY="$IRONIC_RAMDISK_SSH_KEY" \
     --env IRONIC_AGENT_IMAGE="$IRONIC_AGENT_IMAGE" \
-    --env IRONIC_AGENT_PULL_SECRET="{{.PlatformData.BareMetal.PullSecretBase64}}" \
```

Comment:
There were quite a few changes required to get this into the template data that could be reverted now. See: https://github.com/zaneb/openshift-installer/commit/24cf61e62c5371cd9b5817f2d98b36be5a3f33a1

Reply:
Thanks @zaneb I missed the `GetTemplateData` signature, fixed now